test/jdk/java/awt/print/PrinterJob/PageRanges.java

```diff
@@ -23,56 +23,57 @@
 
 /**
  * @test
- * @bug 6575331
+ * @bug 6575331 8320677
  * @key printer
  * @summary The specified pages should be printed.
- * @run main/manual=yesno PageRanges
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual PageRanges
  */
 
-import java.awt.*;
-import java.awt.print.*;
+import java.awt.Graphics;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
 
 public class PageRanges implements Printable {
-
-    static String[] instr = {
-     "This test prints two jobs, and tests that the specified range",
-     "of pages is printed. You must have a printer installed for this test.",
-     "In the first dialog, select a page range of 2 to 3, and press OK",
-     "In the second dialog, select ALL, to print all pages (in total 5 pages).",
-     "Collect the two print outs and confirm the jobs printed correctly",
-    };
+    private static final String INSTRUCTIONS = """
+                "This test prints two jobs, and tests that the specified range",
+                "of pages is printed. You must have a printer installed for this test.",
+                "In the first dialog, select a page range of 2 to 3, and press OK",
+                "In the second dialog, select ALL, to print all pages (in total 5 pages).",
+                "Collect the two print outs and confirm the jobs printed correctly",
+                """;
 
     public static void main(String args[]) throws Exception {
-        for (int i=0;i<instr.length;i++) {
-            System.out.println(instr[i]);
-        }
+        PassFailJFrame passFailJFrame = new PassFailJFrame(INSTRUCTIONS);
+        passFailJFrame.positionTestWindow(null, PassFailJFrame.Position.HORIZONTAL);
```

Comment:
Use `PassFailJFrame.builder()` to configure and create an instance of `PassFailJFrame`.